docs/content/design/coordinator.md

```diff
@@ -66,7 +66,7 @@ To ensure an even distribution of segments across Historical nodes in the cluste
 ### Compacting Segments
 
 Each run, the Druid Coordinator compacts small segments abutting each other. This is useful when you have a lot of small
-segments which may degrade the query performance as well as increasing the disk space usage.
+segments which may degrade the query performance as well as increasing the disk space usage. See [Segment Size Optimization](../operations/segment-optimization.html) for details.
```

Comment:
"segments which may degrade the query performance as well as increasing the disk space usage" -> "segments which may degrade query performance as well as increase disk space usage"

Reply:
Thanks, fixed!